content/docs/gke/deploy/oauth-setup.md

```diff
@@ -48,21 +48,14 @@ address to verify the user's identity.
    * In the **Name** box enter any name for your OAuth client ID. This is *not*
      the name of your application nor the name of your Kubeflow deployment. It's
      just a way to help you identify the OAuth client ID.
-   * In the **Authorized redirect URIs** box, enter the following:
+   * In the **Authorized redirect URIs** box, enter the following (if not already exists):
 
         ```
-        https://<deployment_name>.endpoints.<project>.cloud.goog/_gcp_gatekeeper/authenticate
+        https://iap.googleapis.com/v1/oauth/clientIds/<CLIENT_ID>:handleRedirect
         ```
-        * `<deployment_name>` must be the name of your Kubeflow deployment. It 
-          must have the same value as the deployment name used in the next step 
-          when you deploy Kubeflow from the UI or by running the deployment
-          script. **Deployment name must be 4-20 characters long.**
-          The default value for the deployment name is the `KFAPP` value 
-          used when initializing your Kubeflow app, but you can configure this 
-          with the environment variable `DEPLOYMENT_NAME`.
-        * `<project>` is your GCP project. It must have the same value as used 
-           in the next step when you deploy Kubeflow from the UI or by running 
-           the deployment script.
+        * `<CLIENT_ID>` is the name for your OAuth client ID.
```

Comment:
This statement seems wrong, when comparing it to the screenshot:

```
`<CLIENT_ID>` is the name for your OAuth client ID.
```

Based on line 48 above, the name for the OAuth client ID seems to be a user-nominated, human-readable name. But based on the screenshot, it seems to be a generated string. We should either change the description here, or provide a screenshot with a more readable client name if the description is correct.

Reply:
good catch, thanks! it's not the "name"